doc/tutorial/statistical_inference/model_selection.rst

```diff
@@ -41,40 +41,60 @@ data in *folds* that we use for training and testing::
     >>> print(scores)
     [0.93489148580968284, 0.95659432387312182, 0.93989983305509184]
 
-.. currentmodule:: sklearn.cross_validation
+.. currentmodule:: sklearn.model_selection
 
-This is called a :class:`KFold` cross validation
+This is called a :class:`KFold` cross-validation.
 
 .. _cv_generators_tut:
 
 Cross-validation generators
 =============================
 
+Scikit-learn has a collection of classes which can be used to generate lists of
+train/test indices for popular cross-validation strategies.
 
+The classes are data-independent and are initialized only with parameters that
+characterize the strategy and not the input data. (Like ``k`` in k-fold)
 
-The code above to split data in train and test sets is tedious to write.
-Scikit-learn exposes cross-validation generators to generate list
-of indices for this purpose::
+They expose a :func:`BaseCrossValidator.split` method which accepts the input
```

Comment:
Why do you refer to ``BaseCrossValidatior``? Why should a user care about our class hierarchy? Just say all implement split and here is an example for k-fold.